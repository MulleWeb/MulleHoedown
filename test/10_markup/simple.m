#import <MulleHoedown/MulleHoedown.h>


int   main( void)
{
   printf( "%s\n", [[@"# Headline" hoedownedString] cStringDescription]);
   return( 0);
}
