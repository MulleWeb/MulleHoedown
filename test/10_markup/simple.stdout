<h1 id="Headline">Headline</h1>

